tests/neg/enum-repeated-4470.scala

```diff
@@ -0,0 +1,10 @@
+object RepeatedEnum {
```

Comment:
Our convention is to name our regression test files like `i4470.scala`. If you have mutiple test cases: `i4470a.scala`, `i4470b.scala`

Reply:
Alrighty, will rename the files thanks for the hint.